include/picongpu/param/particle.param

```diff
@@ -56,16 +56,13 @@ namespace picongpu
 
         namespace manipulators
         {
-            /** Define initial particle drift direction vector.
-             *  This vector is used in struct DriftParam.
-             */
-            CONST_VECTOR(float_X, 3, DriftParam_direction, 1.0, 0.0, 0.0);
-
             /** Define Lorentz factor of initial particle drift. */
             struct DriftParam
             {
                 static constexpr float_64 gamma = 1.0;
-                const DriftParam_direction_t direction;
+                /** Define initial particle drift direction vector.
+                 */
+                static constexpr auto driftDirection = float3_X(1.0, 0.0, 0.0);
```

Comment:
in the old code the variable was named `direction` not `driftDirection`.

Reply:
This has also been renamed in `include/picongpu/particles/manipulators/unary/Drift.hpp` :+1: